crates/sqllib/src/dec.rs

```diff
@@ -889,7 +903,7 @@ pub fn power_SqlDecimal_i32(left: SqlDecimal, right: i32) -> F64 {
     let power = SqlDecimal::from(right);
     context.pow(&mut result, &power.get_dec());
     if is_error(&context) {
-        panic!("Error during POWER computation");
+        panic!("Error during computation: power({}, {})", left, right);
```

Comment:
Extract error type from `context`

Reply:
I tried, but I failed to cause an error so I can see what it looks like when printed